pkg/acl/acl.go

```diff
@@ -108,17 +108,21 @@ func (r *Rule) Decode(buf []byte) {
 	r.Mask = rb.Get16()
 	r.Other = rb.Get16()
 	uCnt := rb.Get32()
-	r.NamedUsers = make([]Entry, uCnt)
-	for i := 0; i < int(uCnt); i++ {
-		r.NamedUsers[i].Id = rb.Get32()
-		r.NamedUsers[i].Perm = rb.Get16()
+	if uCnt > 0 {
```

Comment:
If not set, keep it as nil. for json marshal and unmarshal result checking in LoadMeta and DumpMeta